scripts/pods.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+USE_PODS=${FP_PODS:-yes}
+
+if [[ $USE_PODS = 'yes' ]]; then
+	if [[ $(uname) = 'Darwin' ]]; then
+		bundle install --path .bundle
```

Comment:
Is this required? I don't use `.bundle` in my Ruby projects, but if it's necessary in this case then I guess it's necessary.

Reply:
Where does bundler install packages by default? I don’t actually know; this is what my shell autocompletes when I do “bundle install” 😅

If I shouldn’t need this, then I’m happy to take it out.

Reply:
You shouldn't need this; the default installation location is fine.

You can use `gem environment` to figure out where it's installing gems. `~/.gem` would be a place that it'd install.